### PR TITLE
Update radio schedules

### LINKED ISSUE #3296: Live and Next Labels are incorrect in Radio Schedules
**Describe the bug**
Currently, the 'Live' label, and the 'Next' label are always being displayed in English.

Instead of passing in a 'stateLabel' that is part of the schedule data, we should instead just pass all the translations as props, from the service config, and let the component decide which to show based on the state.

It appears that currently in Simorgh, we are passing 'next' or 'live' as the state label to be used for the string that is shown to the user, and uppercasing it in the Psammead component. We should remove this. We should pass the translation strings in as they appear in config, and then render them.

We are also setting visually hidden text incorrectly for when 'Live' is being displayed in english:

Screenshot 2020-03-24 at 15 37 54

This is because there is a check in the Psammead component that the state label is the string 'Live', which it is not going to be. We should again remove this check, and explicitly pass in the visually hidden text.

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to a

**Expected behaviour**
If the programme is live, we show the 'LIVE' label in the service language.
If 'LIVE' is in english, we set this to aria hidden and render the word 'Live' as visually hidden text.
If the programme is on next, we show the 'NEXT' label in the service language.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by tochwill**
Resolves https://github.com/bbc/psammead/issues/3296

**Overall change:** 
- Pulled in newest version of radio schedules component
- Added key for 'NEXT' to translations in service config
- Passed translation strings as props to radio schedules

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [x] This PR requires manual testing

### CODE REVIEW

**Comment by tochwill** [comment]
I should also get rid of stateLabel from the processing function, on that now

**Review by Bopchy — APPROVED** [review]

**Review by EinsteinNjoroge — APPROVED** [review]
👍

**Review by itimirichard — APPROVED** [review]
Thanks Tom. Looks good!

File: src/app/lib/config/services/serbian.js [hunk 1]
```diff
@@ -240,6 +240,7 @@ export const service = {
         listen: 'Slušajte',
         watch: 'Gledajte',
         liveLabel: 'UŽIVO',
+        nextLabel: 'NEXT',
```
Comment (sareh) [inline src/app/lib/config/services/serbian.js:243]: There should also be a label for the other serbian variant.

File: src/app/lib/config/services/ukchina.js [hunk 2]
```diff
@@ -213,6 +213,7 @@ export const service = {
         listen: '收听',
         watch: '观看',
         liveLabel: '直播',
+        nextLabel: 'NEXT',
```
Comment (sareh) [inline src/app/lib/config/services/ukchina.js:216]: There should also be a label for the other ukchina variant.

File: src/app/lib/config/services/zhongwen.js [hunk 3]
```diff
@@ -243,6 +243,7 @@ export const service = {
         listen: '收听',
         watch: '观看',
         liveLabel: '直播',
+        nextLabel: 'NEXT',
```
Comment (sareh) [inline src/app/lib/config/services/zhongwen.js:246]: There should also be a label for the other zhongwen variant.

Comment (tochwill) [reply src/app/lib/config/services/zhongwen.js:246]: Ah, thank you - I'll fix that now

**Review by sareh — APPROVED** [review]

**Comment by EinsteinNjoroge** [comment]
@tochwill A newer version of `psammead-radio-schedules` exists. version `0.1.0-alpha.25.` seeks to solve [this issue](https://github.com/bbc/simorgh/issues/5964) . We should update to that version.

**Comment by PriyaKR** [comment]
Looks good to me
If the programme is live, we show the 'LIVE' label in the service language. - looks good
If 'LIVE' is in english, we set this to aria hidden and render the word 'Live' as visually hidden text.  - looks good
If the programme is on next, we show the 'NEXT' label in the service language. -  looks good

Also the dependency bump.

**Pull request merged**